Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -94,12 +95,14 @@ enum class WaitForOption {
     DispatchIncomingSyncMessagesWhileWaiting = 1 << 1,
 };
 
+#define CONNECTION_STRINGIFY(line) #line
+#define CONNECTION_STRINGIFY_MACRO(line) CONNECTION_STRINGIFY(line)
```

Comment:
Super Nit:  Use something more generic that `line` for the argument like maybe `macro`?

It just so happens the current use is the `__LINE__` macro.

Reply:
This is following the pattern of _STRINGIFY and _XSTRINGIFY, and I think it's fine to keep it "line" when it's only used for __LINE__